File: app/src/main/java/com/sorcerer/sorcery/iconpack/ApplyActivity.java
package com.sorcerer.sorcery.iconpack;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;

import com.sorcerer.sorcery.iconpack.apply.OverviewFragment;
import com.sorcerer.sorcery.iconpack.ui.activities.base.BaseFragmentSubActivity;

/**
 * @description: *
 * @author: Sorcerer
 * *
 * @date: 2016/12/2
 */

public class ApplyActivity extends BaseFragmentSubActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setToolbarBackIndicator();
    }

    @Override
    protected int provideFragmentContainer() {
        return R.id.frameLayout_apply_fragment_container;
    }

    @Override
    protected Fragment provideInitFragment() {
        return new OverviewFragment();
    }

    @Override
    protected int provideLayoutId() {
        return R.layout.activity_apply;
    }
}

File: app/src/main/java/com/sorcerer/sorcery/iconpack/ApplyActivity.kt
package com.sorcerer.sorcery.iconpack

import android.os.Bundle
import android.support.v4.app.Fragment

import com.sorcerer.sorcery.iconpack.apply.OverviewFragment
import com.sorcerer.sorcery.iconpack.ui.activities.base.BaseFragmentSubActivity

/**
 * @description:
 * *
 * @author: Sorcerer
 * *
 * @date: 2016/12/2
 */

class ApplyActivity : BaseFragmentSubActivity() {


    override fun provideLayoutId(): Int {
        return R.layout.activity_apply
    }

    override fun provideFragmentContainer(): Int {
        return R.id.frameLayout_apply_fragment_container
    }

    override fun provideInitFragment(): Fragment {
        return OverviewFragment()
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setToolbarBackIndicator()

    }

}